.gitignore

```diff
@@ -97,3 +98,6 @@ dmypy.json
 
 # vim backup files
 *~
+
+# Notebook temporal files
+notebooks/*/local/**
```

Comment:
@Debskij I would prefer we name this `output` instead of `local` and only include output data. I prefer that models only be saved in the `models` directory, so it is not confusing that models are in multiple directories.

Reply:
Done